go.mod

```diff
@@ -1,19 +1,44 @@
 module github.com/ONSdigital/dp-frontend-homepage-controller
 
-go 1.13
+go 1.17
```

Comment:
Can we go to 1.18 please. 
We will need to bump the docker image to the latest (1.18.3) in [build.yml](https://github.com/ONSdigital/dp-frontend-homepage-controller/blob/develop/ci/build.yml) and [unit.yml](https://github.com/ONSdigital/dp-frontend-homepage-controller/blob/develop/ci/unit.yml) too